rootfs/api/models/app.py

```diff
@@ -33,6 +34,23 @@
 logger = logging.getLogger(__name__)
 
 
+def is_experimental_native_ingress():
```

Comment:
when we can, we should be using `settings.EXPERIMENTAL_NATIVE_INGRESS` instead as seen in some other places. We can still make it [come from an environment variable](https://github.com/deis/controller/blob/a8002a2172bc37801fc062f9a8a9fb65e50f4be3/rootfs/api/settings/production.py#L260), but Django has better support in general for settings than os envvars. The `is_experimental_native_ingress()` function could be removed entirely in lieu of `DEIS_EXPERIMENTAL_NATIVE_INGRESS = bool(os.getenv('EXPERIMENTAL_NATIVE_INGRESS', 0))` in production.py, for example.

Example usage:

https://github.com/deis/controller/blob/a8002a2172bc37801fc062f9a8a9fb65e50f4be3/rootfs/api/settings/production.py#L147
https://github.com/deis/controller/blob/a8002a2172bc37801fc062f9a8a9fb65e50f4be3/rootfs/api/models/app.py#L783
https://github.com/deis/controller/blob/a8002a2172bc37801fc062f9a8a9fb65e50f4be3/rootfs/api/tests/test_app.py#L657-L658